tenant_schemas/postgresql_backend/base.py

```diff
@@ -10,10 +10,7 @@
 from tenant_schemas.utils import get_public_schema_name, get_limit_set_calls
 from tenant_schemas.postgresql_backend.introspection import DatabaseSchemaIntrospection
 
-
-ORIGINAL_BACKEND = getattr(settings, 'ORIGINAL_BACKEND', 'django.db.backends.postgresql_psycopg2')
-# Django 1.9+ takes care to rename the default backend to 'django.db.backends.postgresql'
-original_backend = django.db.utils.load_backend(ORIGINAL_BACKEND)
+original_backend = django.db.utils.load_backend('django.db.backends.postgresql')
```

Comment:
This can cause issues, i can see the need to not break this, i can come up with a couple scenarios on why you would want to override this.